src/components/IdentifierCardEditing.vue

```diff
@@ -113,20 +116,63 @@ export default defineComponent({
             default: 0
         }
     },
+    components: {
+        SchemaGuideLink
+    },
     setup (props) {
+        const linkInfo: linkInfoType = {
+            doi: { linkLabel: 'DOI', linkUrl: '#definitionsdoi' },
+            url: { linkLabel: 'URL', linkUrl: '#definitionsurl' },
+            swh: {
+                linkLabel: 'Software Heritage identifier',
+                linkUrl: '#definitionsswh-identifier'
+            },
+            other: { linkLabel: 'identifier', linkUrl: '#definitionsidentifier-description' }
+        }
+
         return {
             typeOptions: [
                 { label: 'DOI', value: 'doi' },
                 { label: 'URL', value: 'url' },
                 { label: 'Software Heritage', value: 'swh' },
                 { label: 'Other', value: 'other' }
             ],
+            getLinkLabel: computed(() => {
+                // console.log("label props:", props.type);
+                // return "Label"
+                return linkInfo[props.type].linkLabel
+            }),
```

Comment:
```suggestion
            label: computed(() => linkInfo[props.type].label),
```

- renamed exported name
- removed commented code
- compacter version of fat arrow
- updated name of LinkInfo property